packages/pigeon/platform_tests/test_plugin/android/src/main/kotlin/com/example/test_plugin/CoreTests.gen.kt

```diff
@@ -109,22 +109,22 @@ data class AllNullableTypes (
   companion object {
     @Suppress("UNCHECKED_CAST")
     fun fromList(list: List<Any?>): AllNullableTypes {
-      val aNullableBool = list[0] as? Boolean
-      val aNullableInt = list[1].let { if (it is Int) it.toLong() else it as? Long }
-      val aNullableDouble = list[2] as? Double
-      val aNullableByteArray = list[3] as? ByteArray
-      val aNullable4ByteArray = list[4] as? IntArray
-      val aNullable8ByteArray = list[5] as? LongArray
-      val aNullableFloatArray = list[6] as? DoubleArray
-      val aNullableList = list[7] as? List<Any?>
-      val aNullableMap = list[8] as? Map<Any, Any?>
-      val nullableNestedList = list[9] as? List<List<Boolean?>?>
-      val nullableMapWithAnnotations = list[10] as? Map<String?, String?>
-      val nullableMapWithObject = list[11] as? Map<String?, Any?>
-      val aNullableEnum: AnEnum? = (list[12] as? Int)?.let {
+      val aNullableBool = list[0] as Boolean?
+      val aNullableInt = list[1].let { if (it is Int) it.toLong() else it as Long? }
+      val aNullableDouble = list[2] as Double?
+      val aNullableByteArray = list[3] as ByteArray?
+      val aNullable4ByteArray = list[4] as IntArray?
+      val aNullable8ByteArray = list[5] as LongArray?
+      val aNullableFloatArray = list[6] as DoubleArray?
+      val aNullableList = list[7] as List<Any?>?
```

Comment:
i'm curious - in Kotlin, can `Any` already represent a nullable?

Reply:
nope